data/transit/route/bus.json

```
@@ -4347,7 +4330,10 @@
       "locationSet": {"include": ["ie"]},
```

If these are duplicates, you should add matchNames so that existing bus routes/stops with this network value get matched with this entry:
```suggestion
      "locationSet": {"include": ["ie"]},
      "matchNames": ["Bus Éireann Expressway"],
```